### DISCUSSION TITLE
Optionally set timestamp when creating a score

### DISCUSSION THREAD

**Discussion by ghost** [langfuse/langfuse#3194] [Ideas]
### Describe the feature or potential improvement

I am scoring my generations in batches.
At the moment, the timestamp is always at the moment of scoring, while I would like it to be the trace timestamp.
It would be very nice to be able to set the timestamp for the score over the api when creating it.
This request is very similar to #1157 .

**Comment by marcklingen** [comment]
Agree, thanks for raising this, created an issue to track the implementation: https://github.com/langfuse/langfuse/issues/3245
